setup/tue-functions.bash

```diff
@@ -223,89 +226,60 @@ export -f _github_https_or_ssh # otherwise not available in sourced files
 
 function tue-make
 {
-    if [ -n "$TUE_ROS_DISTRO" ] && [ -d "$_TUE_CATKIN_SYSTEM_DIR" ]
+    if [ -n "$TUE_ROS_DISTRO" ] && [ -d "$TUE_SYSTEM_DIR" ]
     then
-        case $(cat "$_TUE_CATKIN_SYSTEM_DIR"/devel/.built_by) in
+        case $(cat "$TUE_SYSTEM_DIR"/devel/.built_by) in
         'catkin_make')
-            catkin_make --directory "$_TUE_CATKIN_SYSTEM_DIR" -DCMAKE_BUILD_TYPE=RelWithDebInfo "$@"
+            echo -e "\e[33mcatkin_make is not supported anymore, use catkin tools\e[0m"
```

Comment:
Still want to have this? I think we can just have a `*` arg for which the `echo` will say the tool is not supported.
Also good to have a `return 1`

Reply:
I don't know what will happen when the file doesn't exist, then it should continue to creating the workspace.